CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 0.3.1 (2019-09-19)
```

Comment:
Are all these changes required for 0.3.1? I think we should include only the bare minimum for the yml file patch.